### PR TITLE
Something to consider.

### PR DESCRIPTION
**Pull request by LaxWasHere**

### CODE REVIEW

File: PlotSquared/src/main/java/com/intellectualcrafters/plot/config/C.java [hunk 1]
```diff
@@ -407,7 +407,7 @@
     /*
      * PlotMe
      */
-    NOT_USING_PLOTME("$2This server uses the $1PlotSquared $2plot management system. Please use the $1/ps $2or $1/p2 $2or $1/plots $2instead", "Errors"),
+    NOT_USING_PLOTME("$2This server uses the far superior $1PlotSquared $2plot management system. Please use the $1/ps $2or $1/p2 $2or $1/plots $2instead", "Errors"),
```
Comment (MattBDev) [inline PlotSquared/src/main/java/com/intellectualcrafters/plot/config/C.java]: In what way? The code is a mess.

**Comment by Sauilitired** [comment]
@MattBDev Really? Should I go onto your repository and bash your plugin? Cause, I have a lot of built up rage towards you, and everything you've ever touched :smiling_imp:

**Comment by boy0001** [comment]
@MattBDev 
Oh, fancy seeing you here!

> In what way?
Functionality, stability and performance.
For more information, see here:
https://github.com/IntellectualSites/PlotSquared/wiki/Why-should-you-switch-to-PlotSquared%3F

> The code is a mess.
I'd like to direct you to your [chunk generator](https://github.com/WorldCretornica/PlotMe-DefaultGenerator/blob/master/src/main/java/com/worldcretornica/plotme/defaultgenerator/DefaultChunkGenerator.java) and point out that you have for loops to `find` where the wall is.

Yes, some code in PlotSquared is messy, but it is always the logical and efficient kind of messy. Take our `messy` plot clearing for example. It is more than 800 times faster (than PlotMe) and it doesn't require custom generators to have their own implementation.

Anywho, forgetting the feature creep, the core classes of PlotSquared are friendly enough...
``` Java
// Clearing a plot
plot.clear(Runnable whenDone);

// inventory gui
PlotInventory inv = new PlotInventory(player, 2, "Plot Jukebox") {
    public boolean onClick(int index) {
        // do click related stuff here
    }
};
inv.setItem(0, new PlotItemStack(7, 0, 1, "PlotMe can't", "do this");
inv.openInventory();

// Player metadata and persistent attributes
plotPlayer.setMeta("blah", someObject);

plotPlayer.setAttribute("titles");
```

..and to be honest, PlotMe has some really weird design decisions; you need a converter just to upgrade the plugin? Plus last I heard, your code was too buggy so you had to pull it from spigot.

**Comment by Sauilitired** [comment]
Oh, and if you need an easier overview, then http://plotsquared.com has a nice chart for your precious eyes to look at :)

**Comment by EvilOlaf** [comment]
I'll skip this when updating the german translation file next time as I think modesty is better than hubris.

**Comment by Sauilitired** [comment]
@EvilOlaf That's very modest of you.

**Pull request merged**